src/icp_v2.cc

```diff
@@ -616,24 +616,6 @@ icpHandleIcpV2(int fd, Ip::Address &from, char *buf, int len)
     }
 }
 
-#ifdef ICP_PKT_DUMP
```

Comment:
Can we leave these changes for now please?
I plan this to become the ICP protocol level-2 display. Now that I am reminded, will followup shortly with a PR doing that.

Reply:
I have opened PR #827 with the modern replacement for this dead code.

Reply:
I have rolled back changes to this file. Resolving.